### PR TITLE
Added temperature status

### PR DESCRIPTION
**Pull request by matraj**
Added the view of the temperature status for P2, P3 and the engine (screenshot below). Updated the test script to add temperature dummy values for this view to pull.

image

TODO:
- Add colour palette
- Update font
- Look into better UI styles

### CODE REVIEW

File: src/components/Temperature.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import '../styles/Temperature.css';
+
+class Temperature extends Component {
+
```
Comment (rageandqq) [inline src/components/Temperature.jsx]: Remove 2 blank spaces.

File: src/styles/Temperature.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.Temperature {
+  color: #fdfdfd;
+  height: 110%;
+  width: 100%;
+  padding: 2px 16px;
+}
+
+.card {
+  /*background: #28292d;*/
```
Comment (rageandqq) [inline src/styles/Temperature.css]: Remove

File: src/styles/Temperature.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  width: 33.33%;
+}
+
+.Temperature h2 {
+  /*color: #333;*/
+}
+
+.accent {
+  background: linear-gradient(to bottom, #70f2de 0%, #469fe9 100%);
+  /*background: linear-gradient(to bottom, #3494E6 0%, #ec6ead 100%);*/
```
Comment (rageandqq) [inline src/styles/Temperature.css]: Remove

File: src/styles/Temperature.css [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  width: 100%;
+  padding: 2px 1px;
+}
+
+.Temperature .column {
+  float: left;
+  width: 33.33%;
+}
+
+.Temperature h2 {
```
Comment (rageandqq) [inline src/styles/Temperature.css]: Remove this style

File: src/components/Temperature.jsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+  render() {
+    return (
+      <div className="Temperature">
+        <div className="card">
```
Comment (rageandqq) [inline src/components/Temperature.jsx]: Change to "Temperature-card"

File: src/components/Temperature.jsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+          <h1>Internal Temperature</h1>
+          <div className="column">
+            <h2>ICE</h2>
+            <span className="accent">{this.props.engTemp} &#8451;</span>
+          </div>
+          <div className="column">
+            <div className="column-center">
```
Comment (rageandqq) [inline src/components/Temperature.jsx]: Why do you need this style?

File: src/components/Temperature.jsx [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+              <h2>P2</h2>
+              <span className="accent">{this.props.p2Temp} &#8451;</span>
```
Comment (rageandqq) [inline src/components/Temperature.jsx]: I'd rename these to something else

File: src/components/Temperature.jsx (hunk 1 above)
Comment (matraj) [reply src/components/Temperature.jsx]: Done

**Pull request merged**